Codec/Picture/Extra.hs

```diff
@@ -130,3 +133,38 @@ rotateRight90 img@Image {..} =
   where
     gen x y = pixelAt img y (imageHeight - 1 - x)
 {-# INLINEABLE rotateRight90 #-}
+
+
+-- | Rotate image by 180, i.e flip both vertically and horizontally.
+
+rotate180 :: Pixel a => Image a -> Image a
+rotate180 img@(Image w h _) = generateImage g w h
+  where
+    g x y = pixelAt img (w - 1 - x) (h - 1 - y)
+{-# INLINEABLE rotate180 #-}
+
+-- | Create an image by placing two images side by side.
+--   If the images are of differnet heights the smaller height is used.
+
+beside :: Pixel a => Image a -> Image a -> Image a
```

Comment:
Sure, I only had use for the two image version but I'll generalize when I get a few minutes.